symphony/bdk/core/service/message/multi_attachments_messages_api.py

```diff
@@ -0,0 +1,214 @@
+from symphony.bdk.gen.agent_api.messages_api import MessagesApi
+from symphony.bdk.gen.agent_model.v4_message import V4Message
+from symphony.bdk.gen.agent_model.v4_message_blast_response import V4MessageBlastResponse
+from symphony.bdk.gen.api_client import Endpoint
+from symphony.bdk.gen.model_utils import (
+    file_type
+)
+
+
+class MultiAttachmentsMessagesApi(MessagesApi):
+
+    def __init__(self, api_client=None):
+        super().__init__(api_client)
+
+        def __v4_message_create_post(self, **kwargs):
+            kwargs['async_req'] = kwargs.get(
```

Comment:
Could you explain why this is needed? I was expecting the default values to be what you put here.

Reply:
Actually I took inspiration from what was generated in the `MessagesApi` class, I refactored a bit but I decided to keep the fields that I'm not sure to be safe: https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/blob/2.0/symphony/bdk/gen/agent_api/messages_api.py#L859-L877

Reply:
👌 for me